### ISSUE TITLE
Sclerosis of the hand tarsal bones

### ISSUE THREAD

**Issue by NuriaQueralt** [obophenotype/human-phenotype-ontology#2323]
>This is a request for a new term

>1. Preferred term label (e.g., Asplenia)_
Sclerosis of the hand tarsal bones

>2. Synonyms (e.g., Absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID to refer to any relevant article that provides additional information about the suggested term).

>4. Reference (PMID)
This reference relates this requested phenotype with the NGLY1-deficiency rare disease
https://www.ncbi.nlm.nih.gov/pubmed/27388694

>5. Suggested parent term 
http://compbio.charite.de/hpoweb/showterm?id=HP:0004054

>6. If you know of a disease that is characterized by this feature, please indicate it (e.g., by its Orphanet or OMIM id), and we will add the annotation to our database.
DO: DOID:0060728
ORPHANET: ORPHA:404454
OMIM:615273

>7. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as HPO:probinson (organization, first initial-last name)
http://orcid.org/0000-0003-0169-8159

**Comment by pnrobinson** [comment]
The tarsal bones are in the feet, not in the hands. The hands have carpal bones, is that what you mean?

**Comment by NuriaQueralt** [comment]
@pnrobinson @nicolevasilevsky @LCCarmody sorry for my delayed response.   the exact sentence i need to annotate is: 'sclerosis of the phalanges or tarsal bones'.  I did not know that tarsal bones are only in the feet.  Then, i presume that i can annotate this using a combination of HP terms: for sclerosis of tarsal bones (HP:0031051), and for sclerosis of the phalanges using both terms for hand and foot phalanges (sclerosis of the finger falanx HP:0100899; sclerosis of the toe phalanx HP:0100924), what do you think?

**Comment by nicolevasilevsky** [comment]
Oops - the new term that was added is incorrect, I will rename it to 'Sclerosis of the hand carpal bones'
@NuriaQueralt I agree with your proposed annotations using sclerosis of tarsal bones (HP:0031051), and for sclerosis of the phalanges using both terms for hand and foot phalanges (sclerosis of the finger falanx HP:0100899; sclerosis of the toe phalanx HP:0100924)

**Comment by nicolevasilevsky** [comment]
@pnrobinson and @drseb 
I realized this term (HP_0500031 _Sclerosis of the carpal bones_) is very similar to HP_0004281 _Irregular sclerosis of hand bones_

However, the logical definitions are different. And now I am noticing the the logical definitions for sclerosis terms are very inconsistent.

Do we want to keep both of these terms? If yes, I think HP_0500031 _Sclerosis of the carpal bones_ should have the layperson synonym 'Increased bone density in hand bones' and we should remove that synonym from HP_0004281 _Irregular sclerosis of hand bones_

**Comment by pnrobinson** [comment]
The two terms are different -- the problem is that the definition for irregular sclerosis of carpal bones refers to 'Sclerosis of hand bones' instead of 'Sclerosis of carpal bones' -- is this what you mean?

Sclerosis of hand bones

'has part' some
    ('increased amount'
     and ('inheres in' some
        (sclerosis
         and ('occurs in' some 'skeleton of manus')))
     and ('has modifier' some abnormal))

Sclerosis of carpal bones

'has part' some
    ('increased amount'
     and ('inheres in' some
        (sclerosis
         and ('occurs in' some 'carpal bone')))
     and ('has modifier' some abnormal))


Irregular sclerosis of carpal bones

'has part' some
    ('Sclerosis of hand bone'
     and ('has modifier' some 'irregular spatial pattern')
     and ('has modifier' some abnormal))



Peter Robinson

Professor of Computational Biology

The Jackson Laboratory for Genomic Medicine

10 Discovery Drive

Farmington, CT 06032

860.837.2095 t | 860.990.3130 m

peter.robinson@jax.org<mailto:peter.robinson@jax.org>

www.jax.org

Robinson lab: https://robinsongroup.github.io/

The Jackson Laboratory: Leading the search for tomorrow's cures

________________________________

**Comment by pnrobinson** [comment]
Please note that the hand contains many more bones than the carpal bones

**Comment by nicolevasilevsky** [comment]
There doesn't seem to be a term for 'Irregular sclerosis of carpal bones', do you think we should add that as a subclass of HP_0500031 'Sclerosis of the carpal bones'?

**Comment by pnrobinson** [comment]
'Irregular sclerosis of carpal bones' already exists: http://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=HP:0004281
